.golangci.yml

```diff
@@ -17,12 +17,14 @@ linters:
   - bidichk
   - bodyclose
   - containedctx
+  - decorder
```

Comment:
Looking at it's default configuration: https://golangci-lint.run/usage/linters/#decorder

Does it check anything in it's current configuration? I enabled everything and there where a lot of findings. Any recommendations on which checks we should use?

Reply:
Removed.